src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -68,6 +68,34 @@ internal static Type GetTypeHelper(Type typeStart, Type[]? genericArgs, IntPtr p
             return type;
         }
 
+        /// <summary>
+        /// Returns a new <see cref="RuntimeTypeHandle"/> object created from a handle to a RuntimeType.
+        /// </summary>
+        /// <param name="value">An IntPtr handle to a RuntimeType to create a <see cref="RuntimeTypeHandle"/> object from.</param>
+        /// <returns>A new <see cref="RuntimeTypeHandle"/> object that corresponds to the value parameter.</returns>
+        public static RuntimeTypeHandle FromIntPtr(IntPtr value) => new RuntimeTypeHandle(Type.GetTypeFromHandleUnsafe(value));
+
+        /// <summary>
+        /// Returns the internal pointer representation of a <see cref="RuntimeTypeHandle"/> object.
+        /// </summary>
+        /// <param name="value">A <see cref="RuntimeTypeHandle"/> object to retrieve an internal pointer representation from.</param>
+        /// <returns>An <see cref="IntPtr"/> object that represents a <see cref="RuntimeTypeHandle"/> object.</returns>
+        public static IntPtr ToIntPtr(RuntimeTypeHandle value) => value.Value;
+
+        /// <summary>
+        /// A new <see cref="RuntimeTypeHandle"/> object created from a handle to a RuntimeType.
+        /// </summary>
+        /// <param name="value">An IntPtr handle to a RuntimeType to create a <see cref="RuntimeTypeHandle"/> object from.</param>
+        /// <returns>A new <see cref="RuntimeTypeHandle"/> object that corresponds to the value parameter.</returns>
+        public static explicit operator RuntimeTypeHandle(IntPtr value) => FromIntPtr(value);
```

Comment:
I am not sure we need the operators that are redundant with the methods. This is very advanced scenario that does not need convenience.

I had people ask me more than once about whether it is better to use the IntPtr operators or the explicit methods on GCHandle. I always tell them to use the explicit methods: The explicit methods make the intent more obvious.